ontology/json-schema.ttl

```diff
@@ -218,7 +218,7 @@
                                                  
                                                  rdfs:subClassOf <https://www.w3.org/2019/wot/json-schema#DataSchema> ;
                                                  
-                                                 rdfs:comment "Metadata specifying that the <code>null</code> data type is used by the data schema (\"type\": \"null\"). If the type of null then it has only one acceptable value, namely null. E.g., it can be used as part of oneOf declaration, where it is used to indicate, that a data schema can also be null."@en .
+                                                 rdfs:comment "Metadata describing data of type <code>null</code>. This <a>Subclass</a> is indicated by the value <code>null</code> assigned to <code>type</code> in <code>DataSchema</code> instances. If the type of <code>null</code> then it has only one acceptable value, namely null. For instances, it can be used as part of <code>oneOf</code> declaration, where it is used to indicate, that a data schema can also be <code>null</code>."@en .
```

Comment:
Problem with last, additional part: If the type of <code>null</code> then ...

This Subclass only has one acceptable value, namely <code>null</code>. It can be used as part of a <code>oneOf</code> declaration, where it is used to indicate, that the data can also be <code>null</code>.